src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="navbar">
+        <ul class="navbar__list">
+          <li>
+            <a
+              href="#"
+              class="navbar__link is-active"
```

Comment:
Fix classes everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
```